src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -113,6 +113,20 @@ protected static String getPopulatedErrorMessage(String messageTemplate, String
                                      null, new Object[] { messageTemplate, userInput, fieldName, errorReason, maxLength });
     }
 
+    protected static String getPopulatedEmptyStringErrorMessage(String messageTemplate, String userInput,
+                                                     String fieldName)
+            throws ReflectiveOperationException {
+        return getPopulatedEmptyStringErrorMessage(messageTemplate, userInput, fieldName, 0);
+    }
+
+    protected static String getPopulatedEmptyStringErrorMessage(String messageTemplate, String userInput,
+                                                     String fieldName, int maxLength)
+            throws ReflectiveOperationException {
+        return (String) invokeMethod(FieldValidator.class, "getPopulatedEmptyStringErrorMessage",
```

Comment:
@leeyimin I didn't get you. Can you explain me in brief what change are you expecting?